src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -257,9 +257,10 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     layout.addCheckbox("Highlight messages redeemed with Channel Points",
                        s.enableRedeemedHighlight);
     layout.addDropdown<QString>(
-        "Timestamp format (a = am/pm)",
+        "Timestamp format (a = am/pm, zzz = miliseconds)",
```

Comment:
:)
```suggestion
        "Timestamp format (a = am/pm, zzz = milliseconds)",
```

Reply:
I am way too used to a Polish version of this word ;p thank you!